src/OXRS_Rack32.cpp

```diff
@@ -11,6 +11,24 @@
 #include <MqttLogger.h>               // For logging
 #include <Adafruit_MCP9808.h>         // For temp sensor
 
+// Macro for converting env vars to strings
+#define STRINGIFY(s) STRINGIFY1(s)
```

Comment:
NOTE: this will perform preprocessor substitutions on the firmware metadata - so if you have ESP8266 or ESP32 in the name they will be substituted for `1` (which I think is the library reference/index).

Bit of a pain, just means we have to name the firmware `OXRS-XXX-Name-ESP-FW`...